ci/ubuntu/setup_env.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+export LC_ALL=C.UTF-8
+
+# Update system and install necessary dependencies
+sudo apt-get update
+sudo apt-get install -y protobuf-compiler
+
+# Install rustup, the Rust toolchain installer
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```

Comment:
we should commit to a rust MSRV in sync with our dependencies, currently the one supported by LDK is 1.41.1 and then a recent compiler version like 1.68.0 or 1.69.0 for dev/testing purposes, though discard the latest one served by rustup iirc ?

Reply:
I made a commit with these lines
```
# Install the minimum supported Rust version (MSRV)
rustup install 1.41.1

# Install a newer version of Rust for development/testing
rustup install 1.68.0
```
hopefully that resolves this ?

Reply:
yes all good that way we’ll have a stable MSRV and current rust version is 1.70.0 so we’ll be able to stick to 1.68.0 for a while